PresentScreenings/Entities/FilmInfo.cs

```diff
@@ -77,17 +77,12 @@ string ScreenedFilmText(ScreenedFilm screenedFilm)
         public void AddScreenedFilm(int filmid, string title, string description)
         {
             var screenedFilm = new ScreenedFilm();
-            screenedFilm.FilmId = filmid;
+            screenedFilm.ScreenedFilmId = filmid;
             screenedFilm.Title = title;
             screenedFilm.Description = description;
             ScreenedFilms.Add(screenedFilm);
         }
 
-        public static void AddNewFilmInfo(int filmId, Film.FilmInfoStatus infoStatus)
-        {
-            CheckAddToFilmInfos(new FilmInfo(filmId, infoStatus));
-        }
-
         public static void CheckAddToFilmInfos(FilmInfo filmInfo)
         {
```

Comment:
Please clean up more in-depth.
`CheckAddFilmInfo()` and `SetFilmInfoStatus()` can go, as well as the setter of property `InfoStatus`.

Reply:
Done.